apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1436,6 +1441,16 @@ get_accounts_transactions_pending_by_pubkey_sut(Id) ->
     Id1 = binary_to_list(Id),
     http_request(Host, get, "accounts/" ++ http_uri:encode(Id1) ++ "/transactions/pending", []).
 
+get_transactions_pending_sut() ->
+    Host = internal_address(),
+    http_request(Host, get, "debug/transactions/pending", []).
+
+delete_tx_from_mempool_sut(Hash) when is_binary(Hash) ->
+    delete_tx_from_mempool_sut(binary_to_list(Hash));
+delete_tx_from_mempool_sut(Hash) when is_list(Hash) ->
+    Host = internal_address(),
+    http_request(Host, delete, "node/operator/mempool/hash/" ++ Hash, []).
+
```

Comment:
What is `sut`?

Reply:
It is a conversation in the `aehttp_integration_SUITE` - all endpoint calls live there and have a `_sut` suffix to their name. My best guess is that sut means _system under test_ but the guy who put the convention in place is no longer in the organziaton.

IMO all endpoints shall be moved to an external helper module.

Reply:
Yes, SUT = System Under Test